data/cncf.yaml

```diff
@@ -1395,6 +1395,12 @@
         - code-lite
       exclude:
         - clomonitor
+    - name: knative-extensions/backstage-plugins
+      url: https://github.com/knative-extensions/backstage-plugins
+      check_sets:
+        - code-lite
+      exclude:
```

Comment:
Thanks @aliok!

In the issue linked to this PR you mentioned `Add this repo to CLOMonitor`, but it's excluded. I just wanted to confirm that this is intentional 🙂 When CLOMonitor is excluded, we'll only use this repository to collect issues for CLOTributor, but it won't be used on CLOMonitor. There are some more repositories in this project in a similar situation.

Reply:
Awesome, no problem 👍